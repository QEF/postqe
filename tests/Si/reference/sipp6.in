 &inputpp
   plot_num=6,
   prefix='Si',
   outdir='./tmp/'
   filplot='ppout6',
/
