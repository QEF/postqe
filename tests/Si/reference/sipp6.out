
     Program POST-PROC v.6.1 starts on 13Oct2017 at 14:19:36 

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
          URL http://www.quantum-espresso.org", 
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote

     Parallel version (MPI), running on     1 processors

   Info: using nr1, nr2, nr3 values from input

   Info: using nr1, nr2, nr3 values from input

     IMPORTANT: XC functional enforced from input :
     Exchange-correlation      = PZ ( 1  1  0  0 0 0)
     Any further DFT definition will be discarded
     Please, verify this is what you really want


     G-vector sticks info
     --------------------
     sticks:   dense  smooth     PW     G-vecs:    dense   smooth      PW
     Sum         349     349    121                 4279     4279     869


     Calling punch_plot, plot_num =   6
     Writing data to file  ppout6
     Message from routine chdens:
     namelist plot not found or invalid, exiting

     POST-PROC    :     0.10s CPU         0.11s WALL


   This run was terminated on:  14:19:36  13Oct2017            

=------------------------------------------------------------------------------=
   JOB DONE.
=------------------------------------------------------------------------------=
